runtime/src/governance/election.rs

```diff
@@ -736,6 +736,8 @@ decl_module! {
         fn reveal(origin, commitment: T::Hash, vote: T::AccountId, salt: Vec<u8>) -> Result {
             let sender = ensure_signed(origin)?;
 
+            ensure!(salt.len() <= 32, "salt too large"); // at most 256 bits salt
```

Comment:
maybe this?
`ensure!(salt.len() == 32, "salt should be 32 characters long");`
or 
`ensure!(salt.len() <= 32, "salt should be not longer than 32 characters");`